structure/extension/MagiKey_FATH.yaml

```diff
@@ -0,0 +1,36 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: structure
+
+uri: https://armidalesoftware.com/issue/tags/FATH.htm
+
+standard tag: 'FATH'
+
+specification:
+  - Father
+  - |
+    Information about the father of an individual in a census.
+
+label: 'Father'
+
+payload: null
+
+substructures:
+  "https://gedcom.io/terms/v7/AGE": "{0:1}"
+  "https://gedcom.io/terms/v7/BIRT": "{0:1}"
+  "https://gedcom.io/terms/v7/INDI-FACT": "{0:1}"
+  "https://gedcom.io/terms/v7/MARR": "{0:1}"
+  "https://gedcom.io/terms/v7/OCCU": "{0:1}"
```

Comment:
Reviewing the spec, I realized this is under-defined: it doesn't tell use what tags are used for these structures, and indeed I don't think our current YAML format can, even though the 7.0 specification allows this. See [#298](https://github.com/FamilySearch/GEDCOM.io/issues/298) for more of my thoughts on this.

Reply:
Updated to use "nonconformant tags" now that that label exists.